### PR TITLE
Fix incorrect dependencies

### PR DESCRIPTION
**Pull request by ritch**
@raymondfeng

### CODE REVIEW

**Comment by raymondfeng** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -7,34 +7,32 @@
...
-    "request": "~2.51.0",
+    "node-uuid": "^1.4.2",
+    "primus": "^2.4.12",
+    "request": "^2.51.0",
     "serve-favicon": "^2.0.1",
-    "shelljs": "^0.3.0",
-    "strong-nginx-controller": "~1.0.0",
-    "strong-pm": "^1.0.0"
+    "strong-nginx-controller": "^1.0.0",
+    "strong-pm": "^1.7.3"
```
Comment (rmg) [inline package.json]: This should either be replaced with strong-mesh-models or changed to `^3.0.0`.

Comment (ritch) [inline package.json]: Changing to 3.0.0

**Comment by ritch** [comment]
test please

**Pull request merged**